bucket/reaper.json

```diff
@@ -33,9 +33,29 @@
     "installer": {
         "script": [
             "Move-Item \"$dir\\`$INSTDIR`$_8_\\*\" -Destination \"$dir\"",
-            "Remove-Item \"$dir\\`$*\", \"$dir\\Uninst*\" -Recurse -Force"
+            "Remove-Item \"$dir\\`$*\", \"$dir\\Uninst*\" -Recurse -Force",
+            "New-Item -ItemType \"file\" -Path \"$dir\" -Name \"reaper.ini\" -Value \"[REAPER]\nwnd_state=0\n\" | Out-Null",
```

Comment:
What does `wnd_state=0` mean?

Reply:
this is the default reaper.ini content if you tick the checkbox "Portable install" while installing without scoop. If this file is empty, reaper.exe behaves like it's not present and creates default files in %APPDATA%/REAPER.